src/app/core/auth-validation.service.ts

```diff
@@ -0,0 +1,123 @@
+import { Injectable } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import { BehaviorSubject, Observable} from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthValidationService {
+
+  public isError = new BehaviorSubject<boolean>(false);
+
+  email: FormControl = new FormControl('', [
+    Validators.required,
+    Validators.email
+  ]);
+  userName: FormControl = new FormControl('', [
```

Comment:
Generally, in the software engineering world I feel like we consider username as one word instead of two.

Reply:
true